pkg/services/store/entity/sqlstash/folder_support.go

```diff
@@ -100,7 +100,7 @@ func (s *sqlEntityServer) updateFolderTree(ctx context.Context, x db.ContextExec
 		itemList = append(itemList, &fi)
 	}
 
-	if err := rows.Close(); err != nil {
+	if err := rows.Err(); err != nil {
```

Comment:
The comment is miss-align with the change : `close rows after listing folder items in namespace`

Reply:
Fixed, thank you!